internal/abi/fastly/types.go

```diff
@@ -278,23 +278,19 @@ const (
 	cacheOverrideTagPCI                  cacheOverrideTag = 0b0000_1000 // $pci
 )
 
-// These limits are configurable for the Rust SDK but constants for the Go one.
-// See https://docs.rs/fastly/latest/fastly/limits/index.html
-
-var (
-	// MaxHeaderNameLen is the header name limit
-	MaxHeaderNameLen = 8192
-	// MaxHeaderValueLen is the header value limit
-	MaxHeaderValueLen = 8192
-	// MaxMethodLen is the method limit
-	MaxMethodLen = 1024
-	// MaxURLLen is the url limit
-	MaxURLLen = 1024
-	// InitialSecretLen is the initial size of the buffer to use for decrypting secrets
-	InitialSecretLen = 1024
-)
+// InitialSecretLen is the initial size of the buffer to use for decrypting secrets
+var InitialSecretLen = 1024
```

Comment:
This is my fault, but I don't see any particular reason why this needs to be user adjustable.  I suggest making this `const initialSecretLen = 1024`.

Reply:
Sure, I will update it.

Reply:
Done in https://github.com/fastly/compute-sdk-go/pull/26/commits/847263430e2b8842b5e66f3b3156709002bb4384